tests/cross/src/test/scala/tests/pc/AutoImportsSuite.scala

```diff
@@ -0,0 +1,119 @@
+package tests.pc
+
+import tests.BaseCodeActionSuite
+import scala.meta.pc.AutoImportsResult
+import scala.meta.internal.metals.CompilerOffsetParams
+import scala.meta.internal.jdk.CollectionConverters._
+import scala.meta.internal.metals.TextEdits
+
+object AutoImportsSuite extends BaseCodeActionSuite {
+
+  check(
+    "basic",
+    """|object A {
+       |  <<Future>>.successful(2)
+       |}
+       |""".stripMargin,
+    """|scala.concurrent
+       |java.util.concurrent
+       |""".stripMargin,
+    compat = Map(
+      "2.11" ->
+        """|scala.concurrent
+           |scala.concurrent.impl
+           |java.util.concurrent
+           |""".stripMargin
+    )
+  )
+
+  checkEdit(
+    "basic-edit",
+    """|package a
+       |
+       |object A {
+       |  <<Future>>.successful(2)
```

Comment:
Could you add a case with a partial import?
For example:
`<<concurrent.Future>>`

Reply:
I don't think that works, since we get a diagnostic on the package name which we can't resolve, it seems. For example:

![image](https://user-images.githubusercontent.com/691940/70630226-86dca500-1c2b-11ea-9688-2fed24c63aaf.png)

hovering on `time` won't yield any quick action (no completions are returned either)

Reply:
this is because the classpath index doesn't support querying packages. It should be an easy fix to implement

Reply:
I imagined. Should we consider that out of scope for this PR?

Reply:
It’s best to implement that as a separate pr. I estimate it’s a simple fix